releaseNotes/SpringCloudAzureAppConfigurationConfig.md

```diff
@@ -6,6 +6,15 @@
 
 [Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Product documentation][docs]
 
+## 1.1.5/1.2.5 - April 27, 2020
+
+* Introduced new interfaces, which allows users to customize client builders for connecting to App Configuration and Key Vault. See this [document](https://github.com/microsoft/spring-cloud-azure/tree/master/spring-cloud-azure-starters/spring-cloud-starter-azure-appconfiguration-config#modifying-connection-client) for more details and examples. [#656](https://github.com/microsoft/spring-cloud-azure/issues/656)
+  * `ConfigurationClientBuilderSetup`
+  * `SecretClientBuilderSetup`
+* Fixed the bug that caused configuration to load partially and refresh to fail when both `/application/` and `/<application_name>/` were used but only one of them loaded.
```

Comment:
How partially? Is there a bug I can find more details?

Reply:
Partially means that if if you had 1 config store that used both /application/ and /application_dev/ if one failed to lead the other could still load. I don't think this bug was ever reported as we saw it as a result of the ms docs issue with key vault.

Reply:
I see. I guess the fix is not to load everything (instead of partial) but to drop whatever that has been loaded. We should make it clear. Does this only happen when fail-fast is false?

Reply:
Yes, it only happened with fail-fast set to false. And the change is that the whole store loads or nothing is loaded.

Reply:
We should make it clear for things like this in the future. Here is what I would suggest this time:

When fail-fast is disabled, an application may only load configuration under one context but not the other from an App Configuration store. The bug is fixed so either all configuration is loaded successfully or nothing is loaded to ensure the consistency of configuration in the application.